internal/pkg/manager/spod/spod_controller.go

```diff
@@ -172,7 +172,10 @@ func (r *ReconcileSPOd) Reconcile(_ context.Context, req reconcile.Request) (rec
 	webhook := bindata.GetWebhook(r.log, r.namespace, spod.Spec.WebhookOpts, image,
 		pullPolicy, caInjectType, spod.Spec.Tolerations, spod.Spec.ImagePullSecrets)
 	metricsService := bindata.GetMetricsService(r.namespace, caInjectType)
-	serviceMonitor := bindata.ServiceMonitor(caInjectType)
+	serviceMonitor, err := bindata.ServiceMonitor(foundDeployment, r.scheme, caInjectType)
```

Comment:
Here's my main question: Does this ownership structure make sense? Or should we maybe make the default spod owned by the operator and the DS and the serviceMonitor both by the spod? (Or spod->ds->serviceMonitor) ?

Reply:
If we're gonna have a service monitor for a DS, and the DS is created via the SPOd objects we might want to consider dedicating a service monitor for the DS itself instead of having a deployment+ds bundle as we do today 😕 

The main question is, do we want to support multiple spod deployments at all?